pkg/helper/flowcollector.go

```diff
@@ -80,6 +80,14 @@ func UseConsolePlugin(spec *flowslatest.FlowCollectorSpec) bool {
 		(spec.ConsolePlugin.Enable == nil || *spec.ConsolePlugin.Enable)
 }
 
+func UseTestConsolePlugin(spec *flowslatest.FlowCollectorSpec) bool {
+	if spec.ConsolePlugin.Advanced != nil {
+		env := spec.ConsolePlugin.Advanced.Env[constants.EnvTestConsole]
+		return env == "true"
```

Comment:
I tend to prefer just checking the correct case here, because it's simpler and (this is subjective)  I don't believe the error brings much value here - but I don't have stronger arguments to give :-)

Reply:
it's more for consistency with eBPF environment variables than anything.

These allow upper case "True" for example